currency_rate_update_VN_VCB/README.rst

```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================================================================================
+Currency Rate Update Joint Stock Commercial Bank for Foreign Trade of Vietnam - Vietcombank service
+===================================================================================================
+
+Download exchange rates automatically from
+Joint Stock Commercial Bank for Foreign Trade of Vietnam - Vietcombank (Vietcombank) service
+
+Installation
+============
+
+To install this module, you need to:
+
+    * clone the branch 11.0 of the repository https://github.com/OCA/l10n-vietnam
+    * add the path to this repository in your configuration (addons-path)
+    * update the module list
+    * search for "Currency Rate Update - VCB" in your addons
+    * install the module
+
+The module depends on currency_rate_update module available on https://github.com/OCA/currency
+
+Usage
+=====
+
+The module supports multi-company currency in two ways:
+
+* when currencies are shared, you can set currency update only on one
+  company
+* when currencies are separated, you can set currency on every company
+  separately
+
+A function field lets you know your currency configuration.
+
+If in multi-company mode, the base currency will be the first company's
+currency found in database.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/l10n-vietnam/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Nicolas Bessi <nicolas.bessi@camptocamp.com>
+* Jean-Baptiste Aubort <jean-baptiste.aubort@camptocamp.com>
+* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+* Grzegorz Grzelak <grzegorz.grzelak@openglobe.pl> (ECB, NBP)
+* Vincent Renaville <vincent.renaville@camptocamp.com>
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com> (Port to V7)
+* Agustin Cruz <openpyme.mx> (BdM)
+* Jacque-Etienne Baudoux <je@bcim.be>
+* Juan Jose Scarafia <jjscarafia@paintballrosario.com.ar>
+* Mathieu Benoi <mathben963@gmail.com>
+* Fekete Mihai <feketemihai@gmail.com> (Port to V8)
+* Dorin Hongu <dhongu@gmail.com> (BNR)
+* Paul McDermott
+* Alexis de Lattre <alexis@via.ecp.fr>
+* Miku Laitinen
+* Assem Bayahi
+* Daniel Dico <ddico@oerp.ca> (BOC)
+* Dmytro Katyukha <firemage.dima@gmail.com>
+* Jesús Ventosinos Mayor <jesus@comunitea.com>
+* Vo Hoang Dat <dat.vh@komit-consulting.com>
```

Comment:
You might want to keep only your name in the list for now. Because i don't think other guys contribute in this module